project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.9
+sbt.version=0.13.16
```

Comment:
The change of sbt version and Scala.js version (below) should be in their own, separate commit.

Reply:
Done and bumped sbt to 1.0.1 instead.